lib/gems/pending/util/miq-logger.rb

```diff
@@ -136,8 +135,8 @@ def addPostRollEvent(name, run_now = false, &blk)
 
       def miqLogHeader(fileName = nil, logData = true)
         # Log an opening message framed in "*" chars
-        @miqFileName = fileName if fileName
-        if logData == true && @miqFileName.nil? == false
+        @miqFileName = fileName
```

Comment:
Yeah you're right...If this gets called more than once it will override it the second time...

The Ruby was killing my eyes...I shouldn't have overstepped :smile: